src/App.tsx

```diff
@@ -5,21 +5,37 @@ import AddNewPlant from "./pages/addNewPlantPage/AddNewPlant.tsx";
 import GlobalStyle from "./theme/globalStyles.ts";
 import { theme } from "./theme/theme.ts";
 import { ThemeProvider } from "styled-components";
+import { FormValues } from "./components/form/Form.types.ts";
+
+const App = () => {
+  const token = localStorage.getItem("formData");
+
+  const plantsCollection: FormValues[] =
+    (token && JSON.parse(localStorage.getItem("formData") || "")) || [];
```

Comment:
duplicated code, this: `JSON.parse(localStorage.getItem("formData")` is already stored as `token`